problems/8.types.js

```diff
@@ -17,8 +17,8 @@
  * @param {number} b - the second number
  * @returns {boolean} - if a and b are both number types, return true
  */
-function bothNumbers() {
-
+const bothNumbers = (a, b) => {
+  return a / 1 === a && b / 1 === b;
 }
```

Comment:
Interesting answer. Was going for the use of typeof but this looks like it would work. Cool.